### ISSUE TITLE
Post_Comments_Skin_Classic exists but not available as interface

### ISSUE THREAD

**Issue by oooh-boi** [elementor/elementor#10846]
Post Comments widget doesn't include any options/settings in the form of the interface, however, the class "Post_Comments_Skin_Classic" DOES EXIST and contains all the necessary options. Why is it kept hidden?

**Comment by isaactchurch** [comment]
I have the same issues. I really wish they would enable customizing this.

**Comment by oooh-boi** [comment]
Can't believe that out of 5.000.000 E users & developers only 2 are missing that functionality...

**Comment by ibqn** [comment]
I miss this functionality as well, this should be at least part of the pro plugin...

**Comment by vojtech-richter** [comment]
I also agree this is BS. I think there should be some easier way to edit this thing. If they have form creating tool why not applicable to this, lol.

**Comment by circleb** [comment]
We need this badly!!
